web/src/components/base/Snackbar.tsx

```diff
@@ -0,0 +1,107 @@
+import CloseIcon from '@mui/icons-material/Close'
+import Alert from '@mui/material/Alert'
+import AlertTitle from '@mui/material/AlertTitle'
+import Button from '@mui/material/Button'
+import IconButton from '@mui/material/IconButton'
+import MUISnackbar from '@mui/material/Snackbar'
+import { styled } from '@mui/material/styles'
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+
+import { ExtendedSendingStatusType, SNACKBAR_AUTO_HIDE_DURATION } from 'shared'
+
+import useDimensions from '../../hooks/useDimensions'
+
+const StyledMuiSnackbar = styled(MUISnackbar)(({ theme }) => ({
+  [theme.breakpoints.down('lg')]: {
+    marginBottom: 50,
+  },
```

Comment:
Is that expected or do we need to move it above the footer?
<img width="400" alt="Image" src="https://github.com/user-attachments/assets/a798b602-d911-4c4e-a0a0-daa349567355" />